http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -61,6 +63,62 @@ paths:
               schema:
                 $ref: "#/components/schemas/LeonardoVersion"
       security: []
+  /api/google/v1/runtime/{googleProject}/{name}:
+    post:
+      summary: Creates a new GCE compute instance in the given project with the given name.
+      description: >
+        * The request is completed without waiting for the initiation of the
+        cluster's creation on the Google side.
+          This reduces the response time compared with the other version.
+        * Default labels clusterName, googleProject, serviceAccount, and notebookExtension cannot be overridden.
+      operationId: createRuntime
+      tags:
+        - runtime
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: name
+          description: runtime name. only lowercase alphanumeric characters, numbers and
+            dashes are allowed in cluster name
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateRuntimeRequest"
+      responses:
+        "202":
+          description: Cluster creation request accepted
+        "400":
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+        "409":
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+        "500":
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
   /api/clusters:
```

Comment:
Can we make `/api/google/v1/runtime…` appear in a separate group than `/api/cluster` in Swagger UI?

Reply:
I think it is in separate group (tag)